packages/gatsby-dev-cli/src/utils/version.js

```diff
@@ -0,0 +1,45 @@
+const path = require(`path`)
+
+const isLocalGatsbySite = () => {
+  try {
+    let { dependencies, devDependencies } = require(path.resolve(
+      `./package.json`
+    ))
+    const inGatsbySite =
+      (dependencies && dependencies.gatsby) ||
+      (devDependencies && devDependencies.gatsby)
+    return inGatsbySite
+  } catch (err) {
+    return false
+  }
+}
+
+const getLocalGatsbyVersion = () => {
```

Comment:
I'm not sure how relevant this is for `gatsby-dev-cli` since the version will likely be an ephemeral timestamp based key every time `gatsby-dev-cli` is used on the project

Reply:
Yeah, it outputs something like Gatsby version: 2.17.11-dev-1573575446933

Reply:
Yeah, doesn't seem _very_ useful in my opinion. Shall we just keep version for the `gatsby-dev-cli` package itself?

Reply:
Added it just to be consistent with gatsby-cli

Reply:
Yeah, that was a fair assumption. Let's remove it!

Reply:
Updated